test/server/dom-shims.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env mocha */
+import '../../lib/isorender/dom-shims';
+
+import {expect} from 'chai';
+
+import {SimpleApp} from '../fixtures/simple-app';
+
+customElements.define(`my-app`, SimpleApp);
+
+describe(`customElements registry`, function() {
+  describe(`.get()`, function() {
+    it(`returns component class`, function() {
+      expect(customElements.get(`my-app`)).to.eql(SimpleApp);
```

Comment:
strict equality should work here, right?
```suggestion
      expect(customElements.get(`my-app`)).to.equal(SimpleApp);
```

Reply:
oh is that what the difference between .eql and .equal was. I assumed .eql was an alias. #TIL

Reply:
`eql` in chai is deep-equals, but afaik we can rely on comparing these by reference